src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -36,6 +36,14 @@ def _is_windows():
     return platform.system() == "Windows"
 
 
+class LaunchMode(Enum):
+    """An enumeration over supported Fluent launch modes."""
+
+    LAUNCH_FLUENT_STANDALONE = 1
+    LAUNCH_FLUENT_PIM = 2
+    LAUNCH_FLUENT_CONTAINER = 3
```

Comment:
```suggestion
    STANDALONE = 1
    PIM = 2
    CONTAINER = 3
```
What would you think about simplifying them like this? 

Makes them easier to use with e.g. `if launch_mode == LaunchMode.PIM` rather than `if launch_mode == LaunchMode.LAUNCH_FLUENT_PIM`